spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/it/DatastoreIntegrationTests.java

```diff
@@ -61,15 +63,30 @@ public static void checkToRun() {
 	}
 
 	@Test
-	public void testRepository() throws InterruptedException {
+	public void testSaveAndDeleteRepository() throws InterruptedException {
 
 		TestEntity testEntityA = new TestEntity();
 		testEntityA.setId("a");
+		testEntityA.setColor("red");
```

Comment:
maybe make id, color and shape constructor parameters to simplify?

Reply:
@dmitry-s changes made PTAL.